src/ralph/accounts/tests.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import six
+
+from django.test import TestCase
+
+from ralph.tests import factories
+from ralph.accounts.ldap import manager_country_attribute_populate
+
+
+try:
+    from ralph.accounts.management.commands.ldap_sync import _truncate_surname
```

Comment:
are you sure this is working? _truncate_surname imo will always be imported - should we check for `ldap_module_exists`?
```
from ralph.accounts.management.commands.ldap_sync import ldap_module_exists
NO_LDAP_MODULE = ldap_module_exists
```